docs/java-rest/high-level/ml/put-data-frame-analytics.asciidoc

```diff
@@ -128,6 +128,12 @@ include-tagged::{doc-tests-file}[{api}-classification]
 <12> The number of top classes (or -1 which denotes all classes) to be reported in the results. Defaults to 2.
 <13> Custom feature processors that will create new features for analysis from the included document
      fields. Note, automatic categorical {ml-docs}/ml-feature-encoding.html[feature encoding] still occurs for all features.
+<14> The alpha regularization parameter. A non-negative double.
+<15> The growth rate of the shrinkage parameter. A double in [0.5, 2.0].
+<16> The soft tree depth limit. A non-negative double.
+<17> The soft tree depth tolerance. Controls how much the soft tree depth limit is respected. A double greater than or equal to 0.01.
+<18> The amount by which to downsample the data for stochastic gradient estimates. A double in (0, 1.0].
```

Comment:
The parenthesis there means 0 is not included. We seem to be using this mathematical notation of ranges in the HLRC docs while we don't do it in the reference API. I think it makes sense we don't use it in the reference API as it assumes some mathematical knowledge. In HLRC it might be ok, as it is addressed to developers, but we could consider also removing their use from there. But if we decide to, I'd rather this PR goes in and then we do it through all of HLRC ML docs.

Reply:
@dimitris-athanasiou Cool, thanks for the explanation. I didn't know this convention. Ignore these two suggestions, please.